src/org/zaproxy/zap/extension/ascanrulesBeta/HPP.java

```diff
@@ -19,14 +19,17 @@
 package org.zaproxy.zap.extension.ascanrulesBeta;
 
 import java.io.StringReader;
+import java.net.UnknownHostException;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.TreeSet;
+
 import net.htmlparser.jericho.Element;
 import net.htmlparser.jericho.HTMLElementName;
 import net.htmlparser.jericho.Source;
+
```

Comment:
I know formatting changes aren't supposed to be included. Eclipse handily added these two blank lines when it auto inserted the java.net.UnknownHostException import, I didn't notice them until after I did the PR. If you want a fixup to remove them then just let me know...